src/server/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp

```diff
@@ -191,6 +189,15 @@ struct boss_shade_of_aran : public BossAI
         }
     }
 
+    bool CheckDamageDuringDrinking(uint32 oldHealth)
+    {
+        if (me->GetHealth() < oldHealth)
+        {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    bool CheckDamageDuringDrinking(uint32 oldHealth)
    {
            return me->GetHealth() < oldHealth;
    }
```

Reply:
This function is only used once as far as I can see, check if using a function is really needed here